### ISSUE TITLE
[Accessibility] Add examples of *automatic* accessible name and PosInSet

### ISSUE THREAD

**Issue by citelao** [microsoft/WinUI-Gallery#1227]
### Issue type

sample request

### Which version of the app?

WinUI 3 Gallery

### Description

The new Screen reader documentation has examples of manual accessible names and PosInSet, but does not have examples where XAML provides this for free. We should document that so folks don't think they have to add these properties manually.

**Comment by citelao** [comment]
(Also applies to `TabIndex` and Keyboard Shortcuts in "Keyboard support")
